artview/components/select_region.py

```diff
@@ -13,7 +14,7 @@
 
 from ..core import (Variable, Component, common, VariableChoose,
                     componentsList, QtGui, QtCore)
-from ..core.points import Points, write_points_csv, read_points_csv
+from ..core.points import Points
 
```

Comment:
I wouldn't like a button in SelectRegion, I was thinking more about a "extract points" mode that open and link SelectRegion and PointsDisplay.

If we want to merge it before PointsDisplay is ready, we could allow two parallel SelectRegion, I have the old file as back up.